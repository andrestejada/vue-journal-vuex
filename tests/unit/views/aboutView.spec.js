import About from '../../../src/views/About.vue'
import { shallowMount } from '@vue/test-utils';
describe('testing view about', () => {

  const wrapper = shallowMount(About)
  it('should render correctly', () => {
    expect(wrapper.get('h1').text()).toBe('This is an about page')
    expect(wrapper.html()).toMatchSnapshot()   
  });
});