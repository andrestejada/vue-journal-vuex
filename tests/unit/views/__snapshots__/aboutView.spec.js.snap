// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing view about should render correctly 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
